src/main/java/com/squareup/javapoet/Util.java

```diff
@@ -109,4 +110,19 @@ public static void requireExactlyOneOf(Set<Modifier> modifiers, Modifier... mutu
   public static boolean hasDefaultModifier(Collection<Modifier> modifiers) {
     return DEFAULT != null && modifiers.contains(DEFAULT);
   }
+
+  public static String identifier(String part) {
```

Comment:
Maybe rename to identifierPrefix? Should this Just Work for any string, and return null if it doesn't start with an identifier character? That might make the calling code simpler.